pkg/cmd/init.go

```diff
@@ -138,7 +138,7 @@ func initCerts(cfg *config.MicroshiftConfig) ([]byte, *cryptomaterial.Certificat
 		cryptomaterial.NewCertificateSigner(
 			"kubelet-signer",
 			cryptomaterial.KubeletCSRSignerSignerCertDir(certsDir),
-			cryptomaterial.ClientCAValidityDays,
+			cryptomaterial.KubeControllerManagerCSRSignerSignerCAValidityDays,
```

Comment:
Why do we need the kube-controller-manager-csr-signer-signer if this is used by the kube-controller-manager-operator which MicroShift does not use or ship?

Reply:
I am in fact only using this to stick to the OpenShift schema.

From what I can see, this certificate signs two others:
- the csr-signer CA certificate
- the kubelet client certificates during bootstrap

I think that we may in fact be able to live without this certificate in Microshift, for now I would stick with the current OCP schema until I figure out whether there is any use for this `signer -> intermediate` scheme here.